index.html

```diff
@@ -1,19 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <img src="img/karma-logo.svg" alt="karma-logo" class="logo" />
+    <div class="content">
+      <div class="navbar">
+        <a style="font-weight: bold" href="#">Meet Karma</a>
+        <a href="#">How it Works</a>
+        <a href="#">Store</a>
+        <a href="#">Blog</a>
+        <a href="#">Help</a>
+        <a href="#">Login</a>
+      </div>
+      <!--end div navbar-->
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+      <div class="hero">
+        <p class="hero_paragraph">Introducing Karma</p>
+        <p class="text">Bring WiFi with you, everywhere you go</p>
 
-</body>
+        <!--end div text-->
+        <button class="button">Learn More</button>
+      </div>
+      <!--end div hero-->
+    </div>
+    <!--end div content-->
+    <p class="main_paragraph">Everyone needs a little Karma.</p>
+
+    <container class="container">
+      <div class="img">
```

Comment:
You could make this element a flex container as well as the container class on line 44 to centre the image and the text below it. 
Try this css: 
```
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
```